### PR TITLE
Issue/#9b  create and use mobile menu styled components

### PR DESCRIPTION
**Pull request by thecodeclimber**

### CODE REVIEW

File: src/components/mobile/MobileMenu.tsx [hunk 1]
```diff
@@ -24,9 +36,11 @@ function MobileMenu() {
...
     const conveyorRef = useRef<IMobileMenuConveyorController>(null);
 
-    const rootClasses = classNames('mobile-menu', {
-        'mobile-menu--open': mobileMenu.open,
-    });
+    // const rootClasses = classNames('mobile-menu', {
+    //     'mobile-menu--open': mobileMenu.open,
+    // });
+
+    const isMobileMenuOpen = mobileMenu.open
```
Comment (thecodeclimber) [inline src/components/mobile/MobileMenu.tsx:43]: @lokesh75way 

Please note this code when I discuss the `isOpen` prop below.

File: src/components/mobile/MobileMenu.tsx [hunk 2]
```diff
@@ -41,33 +55,33 @@ function MobileMenu() {
     };
 
     return (
-        <div className={rootClasses}>
-            <div className="mobile-menu__backdrop" onClick={mobileMenuClose} />
-            <div className="mobile-menu__body" ref={bodyRef} onTransitionEnd={onTransitionEnd}>
-                <button className="mobile-menu__close" type="button" onClick={mobileMenuClose}>
+        <MobileMenuBase isOpen={isMobileMenuOpen}>
```
Comment (thecodeclimber) [inline src/components/mobile/MobileMenu.tsx:58]: @lokesh75way 

Please check that `isOpen={isMobileMenuOpen}` will work.  Please see code above where I commented out  `const rootClasses...` and created `const isMobileMenuOpen...`

**Review by thecodeclimber — COMMENTED** [review]
@lokesh75way Please note these comments and make any relevant changes.

**Comment by lokesh75way** [comment]
Github commit link to verify code: https://github.com/thecodeclimber/bluejay-redparts/pull/25/commits/99540f4659217bb952a3f094f77901e44901b845
PR link: https://github.com/thecodeclimber/bluejay-redparts/pull/25
@thecodeclimber

**Pull request merged**